tests/integration/fork/conftest.py

```diff
@@ -11,7 +11,7 @@
 @pytest.fixture(scope="package", autouse=True)
 def forked_env():
     with boa.swap_env(Env()):
-        fork_uri = os.environ.get("MAINNET_ENDPOINT", "http://localhost:8545")
+        fork_uri = os.environ.get("MAINNET_ENDPOINT", "http://localhost:9090")
```

Comment:
please revert this - if you are going to customize the fork endpoint please do it in your local environment!

Reply:
Oops.

Reply:
fixed!